### PR TITLE
The readme should specify the interface

### PR DESCRIPTION
**Pull request by mcollina**
As titled.

### CODE REVIEW

**Comment by delvedor** [comment]
I'm not super sure about this, since this situation changes based on the environment.
For example you can run our examples in your machine without the need to specify an address.

**Comment by jsumners** [comment]
I agree with @delvedor. Maybe add a note like:

> ## Note
> `.listen` binds to the local host, `127.0.0.1`, interface by default. See [the documentation](https://github.com/fastify/fastify/blob/master/docs/Server-Methods.md#listen) for more information.

But, really, it's already documented. People just have to read it.

**Comment by nwoltman** [comment]
I would be good with @jsumners suggestion if there was an explanation somewhere about when to use `0.0.0.0`.

**Comment by mcollina** [comment]
Updated.

File: README.md [hunk 1]
```diff
@@ -52,14 +52,18 @@ fastify.get('/', async (request, reply) => {
...
   if (err) throw err
   console.log(`server listening on ${fastify.server.address().port}`)
 })
 ```
 
 Do you want to know more? Head to the <a href="https://github.com/fastify/fastify/blob/master/docs/Getting-Started.md"><code><b>Getting Started</b></code></a>.
 
+#### Note
+
+`.listen` binds to the local host, `127.0.0.1`, interface by default. See [the documentation](https://github.com/fastify/fastify/blob/master/docs/Server-Methods.md#listen) for more information.# Note
```
Comment (jsumners) [inline README.md]: You have a wayward trailing `# Note` on this line.

**Comment by mcollina** [comment]
Updated

**Pull request merged**